Makefile

```diff
@@ -27,6 +27,16 @@ quotelint:
 		exit 1;                                                       \
 	fi >&2
 
+check-copyright:
+	@found="";                                                                              \
```

Comment:
what's found for?

Reply:
because this went through a bunch of iterations, starting with a for loop that used 'found' to decide when to print the "following files" header. I just missed it when cleaning up.